pyproject.toml

```diff
@@ -18,6 +18,17 @@ isort = "^5.13.2"
 pathos = "^0.3.2"
 pyright ="^1.1.316"
 pytest-mock = "^3.14.0"
+numpy = "^1.26.0"
+geopandas = "^0.14.0"
+pyproj = "^3.6.0"
+fiona = "^1.9.0"
+pyyaml = "^6.0.2"
+typing-extensions = "*"  # Add this for typing.List support
```

Comment:
hm? `typing.List` shouldn't be needed since you can just use `list`, also I don't think this is needed for `typing.List` to work?